doc/classes/Viewport.xml

```diff
@@ -117,6 +117,12 @@
 				Returns the visible rectangle in global screen coordinates.
 			</description>
 		</method>
+		<method name="gui_cancel_drag" qualifiers="const">
+			<return type="void" />
+			<description>
+				Cancels a drag operation that was previously started with [method Control.force_drag].
```

Comment:
This cancels any drag, not only from force_drag.

Reply:
Good call.